module/zstd/zfs_zstd.c

```diff
@@ -764,7 +766,7 @@ module_exit(zstd_fini);
 
 ZFS_MODULE_DESCRIPTION("ZSTD Compression for ZFS");
 ZFS_MODULE_LICENSE("Dual BSD/GPL");
-ZFS_MODULE_VERSION(ZSTD_VERSION_STRING);
+ZFS_MODULE_VERSION(ZSTD_VERSION_STRING "a");
```

Comment:
No, not for debugging. 

While working on this, I saw a number of times where DKMS explicitly printed "hey the version's the same, so I don't have to replace the module" before I started just using the kmod targets.

Plus, even ignoring DKMS doing that, if we get reports of this problem happening in distro packages, not being able to check if this was applied short of going and fetching the source seems unnecessarily difficult.

Reply:
I'd rather keep it, if only so that if someone cherrypicks this and doesn't change the version they tell DKMS, there's no chance it'll keep the old zstd.ko.

Obviously there's no need to keep it if the version bumps in the future, though.

Reply:
...come to think of it, why does it make sense for zzstd.ko to have the zstd version number? It's not like it's going to get updated independently of the rest of the codebase or used by not-ZFS consumers, and as in this example, it probably still makes sense to encode the version the rest of the codebase is using in the string even if we want to expose the zstd version somewhere?

Oh well, a question for a future PR.